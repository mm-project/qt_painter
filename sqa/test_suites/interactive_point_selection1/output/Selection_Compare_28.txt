Name: Selection
ObjCount: 15
======
Rectangle:
Ellipse:
Ellipse:
Polygon:
Polygon:
Polygon:
Polygon:
Rectangle:
Line:
Line:
Line:
Line:
Line:
Line:
Line:
--------