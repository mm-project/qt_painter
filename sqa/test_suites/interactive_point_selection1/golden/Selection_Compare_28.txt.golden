Name: Selection
ObjCount: 13
======
Ellipse:
Ellipse:
Polygon:
Polygon:
Polygon:
Rectangle:
Line:
Line:
Line:
Line:
Line:
Line:
Line:
--------